src/aap_eda/api/serializers/project.py

```diff
@@ -48,6 +55,14 @@ class Meta:
             *read_only_fields,
         ]
 
+    def get_proxy(self, obj: models.Project) -> str:
```

Comment:
The same function appears twice, should be try to avoid duplicated code? maybe through a mixin?